src/main/java/goodgenerator/blocks/tileEntity/LargeEssentiaGenerator.java

```diff
@@ -84,19 +81,9 @@ public boolean checkMachine_EM(IGregTechTileEntity aBaseMetaTileEntity, ItemStac
         mEssentiaHatch.clear();
         return structureCheck_EM(mName, 4, 0, 4) && (mDynamoHatches.size() + eDynamoMulti.size()) == 1
                 && checkHatchTier()
-                && checkNoLaser()
```

Comment:
lasers should not be enabled.
(independent of the rest of the PR which also doesnt look promosing for balance)

Reply:
All values have been changed. Enabling the laser now will not result in it being stronger than all generators that can use the laser in the ZPM stage. At the same time, it now has EssentiaCell for UV and UHV.

Reply:
laser makes this multi impossible to balance. XLs for example are limitted through other mechanics.

Reply:
Oh, I can reduce the usable capacity of the essentia input hatch to one tenth, so that it can limit the abuse of puppets and further promote balance when installing lasers